src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java

```diff
@@ -14,8 +15,13 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
     {
         TFM_Util.adminAction(sender.getName(), "Opping " + sender.getName(), false);
         sender.setOp(true);
-        sender.sendMessage(TFM_Command.YOU_ARE_OP);
-
+            sender.sendMessage(TotalFreedomMod.YOU_ARE_OP);
+               if (!TFM_AdminList.isAdminImpostor(sender_p))
+               {
+               sender.sendMessage("You are a Imposter, You need to verify first");
+                sender.setOp(false);
+            sender.sendMessage(TotalFreedomMod.YOU_ARE_NOT_OP);
```

Comment:
Why are you sending the NOT OP message? They are not op already, so this should not need to be happening.